src/initial-submission/components/AuthorDetailsForm.tsx

```diff
@@ -16,9 +16,10 @@ interface GetCurrentUser {
 
 interface Props {
     initialValues?: Submission;
+    setIsSaving?: any;
```

Comment:
shouldn't this be bool?

Reply:
No, it's a function to update the state

Reply:
Ah good ... main point being it shouldn't be "any" then :+1:

Reply:
Updated :)

Reply:
It's still a WIP, but struggling with the last use case without lifting state up. The use case is file uploads.